### PR TITLE
[WIP] chore: Travis CI on Node.js 12

### PR DESCRIPTION
**Pull request by brodybits**
with extra blank lines

### CODE REVIEW

**Comment by brodybits** [comment]
Any ideas how switching to Node.js 12 can cause a build failure on eslint?

**Comment by breautek** [comment]
Seems a package called [natives](https://www.npmjs.com/package/natives) that is crashing, and it is deprecated relying on nodejs internals.

**Comment by brodybits** [comment]
Now a problem in gulp@4, wonder if we have to update the other gulp dependencies. Adding the **help wanted** label for noiw.

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "classnames": "^2.1.2",
     "envify": "^3.4.0",
     "fs-extra": "^0.11.1",
-    "gulp": "^3.9.0",
+    "gulp": "^4",
```
Comment (timbru31) [inline package.json]: please use `^4.0.2`

**Pull request closed without merging**